src/test/java/bio/terra/aws/resource/discovery/LandingZoneTest.java

```diff
@@ -66,10 +68,34 @@ public void inequality() {
     KmsKey kmsKey = landingZone.getKmsKey();
     StorageBucket storageBucket = landingZone.getStorageBucket();
 
+    // Different App VPC ID
```

Comment:
nit (outside of your changes): I find it a little odd that given all these inequality tests, there's not an equality check against a different instance of a landing zone with the same attributes.

Reply:
I think the check in the `equality` test on line 45/47 does this, but I might be mis-reading?